lib/python/rosie/svn_pre_commit.py

```diff
@@ -65,6 +65,22 @@ def _get_access_info(self, info_node):
         access_list.sort()
         return owner, access_list
 
+    def _get_author_aliases(self, repos, txn=None):
+        """Return the current author alternative name map."""
+        txn_args = ()
```

Comment:
No need to support `txn` here but I suppose it's harmless

Reply:
You're right - it's left over from when I was doing parseability testing for the pre-commit, which I dropped when we went to the simple file in ROSIE. I guess we don't need this any more?

Reply:
Removed